README.md

```
@@ -1 +1,167 @@
+<aside>
+
+
+# 🔬 트러블슈팅
+
+</aside>
+
+- ## Running-Record 생성 API 처리량 개선
+- ## AWS 환경에 배포 후 Spring Cloud Gateway 가 Eureka Client 를 찾지 못하는 문제
+- ## 사용자 / 인증인가 서버분리와 게이트웨이 순환참조 오류
+
+<aside>
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: aside

(MD033, no-inline-html)